test/truffle/integration/tracing/binding.trace

```diff
@@ -7,3 +7,12 @@
 ["line", "/binding.rb", 27, nil, {:captured_in_scope=>2, :inside_trace=>4, :modified=>5, :outside_trace=>1, :result=>nil}, nil]
 ["line", "/binding.rb", 30, nil, {:captured_in_scope=>2, :inside_trace=>4, :modified=>5, :outside_trace=>1, :result=>16}, nil]
 ["line", "/binding.rb", 32, nil, {:captured_in_scope=>2, :inside_trace=>4, :modified=>5, :outside_trace=>1, :result=>16}, nil]
+["call", "/posix.rb", 76, :[]=, {:index=>nil, :length=>nil, :value=>nil}, Array]
```

Comment:
What is this posix call doing in the trace?

Reply:
I see `set_trace_func` activated and disabled in individual files so this should not happen.